package.json

```diff
@@ -104,6 +104,7 @@
     "name": "uiManagers"
   },
   "dependencies": {
-    "@playkit-js/common": "1.5.18"
+    "@playkit-js/common": "1.5.18",
+    "preact-render-to-string": "^6.5.11"
```

Comment:
please use an exact (numeric) version
- update to let's say 6.5.11
- run yarn install
- commit package.json and yarn.lock